src/components/registration/RegistrationTable.jsx

```diff
@@ -1,27 +1,89 @@
-import React from "react";
+import React, { useContext } from "react";
 import MUIDataTable from "mui-datatables";
 import PropTypes from "prop-types";
+import { FieldsContext } from "../../context/fields";
+import QuestionTypes from "../../constants";
 
-const columns = [
+const options = {
+  responsive: "standard",
+  selectableRows: "none",
+};
+
+const defaultColumns = [
+  {
+    name: "first_name",
+    label: "First Name",
+    options: { filter: false },
+  },
+  { name: "last_name", label: "Last Name", options: { filter: false } },
   { name: "email", label: "Email", options: { filter: false } },
   { name: "phone_number", label: "Phone Number", options: { filter: false } },
+  {
+    name: "num_children",
+    label: "# of Children",
+    options: { searchable: false, filterOptions: { fullWidth: true } },
+  },
   {
     name: "preferred_comms",
-    label: "Preferred Communication Method",
-    options: { searchable: false },
+    label: "Preferred Contact",
+    options: { searchable: false, filterOptions: { fullWidth: true } },
   },
 ];
 
-const options = {
-  selectableRows: "none",
-};
+function getTableRows(families, extraFields) {
+  if (extraFields.length) {
+    const fieldMap = Object.assign(
+      {},
+      ...extraFields.map((field) => ({ [field.id]: field.name }))
+    );
+    families.forEach((family) => {
+      const familyRow = family;
+      if (family.parent !== null && family.parent !== undefined) {
```

Comment:
```suggestion
      if (family.parent && family.parent !== undefined) {
```
Let me know if we can't do the conditional this way!